src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -233,6 +237,29 @@ export class NotebookModel extends Disposable implements INotebookModel {
 
 	public set trustedMode(isTrusted: boolean) {
 		this._trustedMode = isTrusted;
+		let whitelistedFolders: string[] = this.workspaceContextService.getValue('notebook.trustedNotebooks');
```

Comment:
@chlafreniere can chime in - but in my opinion this should not be something that is enabled by default and unable to be turned off. Whether to whitelist an entire directory is not something I imagine is a common action and so I would rather have this be an action a user specifically does. (Possibly with a setting to automatically trust all notebooks in the same folder as one the user is trusting that's false by default)